deployment/tf/modules/yc/allocator.tf

```diff
@@ -54,8 +53,8 @@ resource "kubernetes_stateful_set" "allocator" {
             host_port      = local.allocator-port
           }
           port {
-            container_port = 17080
-            host_port = 17080
+            container_port = local.allocator-metrics-port
```

Comment:
nit: It'd be nice to have the same monitoring settings for all services

Literally today I added prometheus monitoring to the LzyService :)

Reply:
Will do in the next pull request. Right now ports in this PR are moved only for convenience. I'll make a solution to add such settings and also run all services on the same node without ports conflict.